src/tasks/appveyor.js

```diff
@@ -0,0 +1,54 @@
+const { yaml } = require('mrm-core');
+
+module.exports = (config) => {
+  // appveyor.yml
+  yaml('appveyor.yml')
+    // Overwrite
+    .set({
+      branches: {
+        only: [
+          'master',
+        ],
+      },
+      init: [
+        'git config --global core.autocrlf input',
+      ],
+      environment: {
+        matrix: [
+          {
+            nodejs_version: '7',
```

Comment:
@michael-ciniawsky - Now configured to use the config for it's nodejs versions like the travis template.